diff --git a/build.gradle b/build.gradle
index a76a5fa60..5c9bdbf3a 100644
--- a/build.gradle
+++ b/build.gradle
@@ -47,14 +47,6 @@
 
     // ========================== COMPILE ==========================
 
-    // https://docs.gradle.org/current/userguide/toolchains.html
-    // gradlew -q javaToolchains - see the list of detected toolchains.
-    java {
-        toolchain {
-            languageVersion = JavaLanguageVersion.of(22)
-        }
-    }
-
     // Append options for unchecked/deprecation
     tasks.withType(JavaCompile).configureEach {
         options.compilerArgs << '-Xlint:unchecked' << '-Xlint:deprecation' << '-g' << '-parameters'
