diff --git a/build.gradle b/build.gradle
index 9efb806a2..488b59afd 100644
--- a/build.gradle
+++ b/build.gradle
@@ -50,14 +50,6 @@ implementation fileTree(dir: "$rootProject.projectDir/libs", include: ['*.jar'])
 
     // ========================== COMPILE ==========================
 
-    // https://docs.gradle.org/current/userguide/toolchains.html
-    // gradlew -q javaToolchains - see the list of detected toolchains.
-    java {
-        toolchain {
-            languageVersion = JavaLanguageVersion.of(System.getenv('TARGET_VERSION') ?: '22')
-        }
-    }
-
     // Append options for unchecked/deprecation
     tasks.withType(JavaCompile).configureEach {
         options.compilerArgs << '-Xlint:unchecked' << '-Xlint:deprecation' << '-g' << '-parameters'
