diff --git a/recaf-ui/build.gradle b/recaf-ui/build.gradle
index 3ad84eca5..cd32ad73c 100644
--- a/recaf-ui/build.gradle
+++ b/recaf-ui/build.gradle
@@ -17,7 +17,6 @@ implementation project(':recaf-core')
     testImplementation(testFixtures(project(":recaf-core")))
 
     implementation(libs.atlantafx)
-    implementation(libs.docking)
     implementation(libs.bundles.ikonli)
     implementation(libs.bundles.image.io)
     implementation(libs.jsvg)
@@ -41,4 +40,4 @@ implementation project(':recaf-core')
     dependencies {
         exclude(dependency(javaFxIncludeInDist ? 'invalid:invalid:invalid' : 'org.openjfx:.*:.*'))
     }
-}
\ No newline at end of file
+}
