diff --git a/build.gradle b/build.gradle
index 135dba8e1..e8eb42df5 100644
--- a/build.gradle
+++ b/build.gradle
@@ -50,14 +50,6 @@ implementation fileTree(dir: "$rootProject.projectDir/libs", include: ['*.jar'])
 
     // ========================== COMPILE ==========================
 
-    // https://docs.gradle.org/current/userguide/toolchains.html
-    // gradlew -q javaToolchains - see the list of detected toolchains.
-    java {
-        toolchain {
-            languageVersion = JavaLanguageVersion.of(22)
-        }
-    }
-
     // Append options for unchecked/deprecation
     tasks.withType(JavaCompile).configureEach {
         options.compilerArgs << '-Xlint:unchecked' << '-Xlint:deprecation' << '-g' << '-parameters'
@@ -146,10 +138,6 @@ implementation fileTree(dir: "$rootProject.projectDir/libs", include: ['*.jar'])
 // You can skip tests by specifying '-x test' in your gradle task arguments.
 tasks.register('test') {
     dependsOn(subprojects.test)
-    doLast {
-        if (subprojects.test.stream().anyMatch(Task::getDidWork))
-            buildJacocoAggregate.execute()
-    }
 }
 
 tasks.register('build') {
