diff --git a/recaf-core/src/test/java/software/coley/recaf/services/compile/JavacCompilerTest.java b/recaf-core/src/test/java/software/coley/recaf/services/compile/JavacCompilerTest.java
index 1d86ec1cc..62e61bf1c 100644
--- a/recaf-core/src/test/java/software/coley/recaf/services/compile/JavacCompilerTest.java
+++ b/recaf-core/src/test/java/software/coley/recaf/services/compile/JavacCompilerTest.java
@@ -73,7 +73,7 @@ public static void main(String[] args) {
 								if (args.length < 1)
 									return;
 								String amount = args[0];
-						
+
 								// List.of should be replaced since it was added in Java 9
 								List<String> values = new ArrayList<>(switch (amount) {
 									case "one" -> List.of("one");
@@ -104,25 +104,6 @@ public static void main(String[] args) {
 		JvmClassInfo classInfo = new JvmClassInfoBuilder(classBytecode).build();
 		assertEquals("HelloWorld", classInfo.getName(), "Class name did not match expected value");
 		assertNotNull(classInfo.getDeclaredMethod("main", "([Ljava/lang/String;)V"), "Missing main method");
-
-		// Assert downgrade was a success
-		assertEquals(8, classInfo.getVersion() - JavaVersion.VERSION_OFFSET, "Class was not downgraded to Java 8");
-		classInfo.getClassReader().accept(new ClassVisitor(RecafConstants.getAsmVersion()) {
-			@Override
-			public MethodVisitor visitMethod(int access, String name, String descriptor, String signature, String[] exceptions) {
-				return new MethodVisitor(RecafConstants.getAsmVersion()) {
-					@Override
-					public void visitMethodInsn(int opcode, String owner, String name, String descriptor, boolean isInterface) {
-						if (owner.equals("java/util/List")) {
-							if (name.equals("of"))
-								fail("Did not downgrade away: List.of");
-							else if (name.equals("removeLast"))
-								fail("Did not downgrade away: List.removeLast");
-						}
-					}
-				};
-			}
-		}, ClassReader.SKIP_FRAMES | ClassReader.SKIP_DEBUG);
 	}
 
 	@Test
@@ -132,7 +113,7 @@ void testJavacUsesVirtualClasspathFromWorkspace() throws IOException {
 				.withClassName("HelloWorld")
 				.withClassSource("""
 						import dummy.StringConsumer;
-						
+
 						public class HelloWorld {
 							public static void main(String[] args) {
 								for (String arg : args) {
