diff --git a/recaf-ui/src/main/java/software/coley/recaf/util/SVG.java b/recaf-ui/src/main/java/software/coley/recaf/util/SVG.java
index b8915bb50..97af8ee99 100644
--- a/recaf-ui/src/main/java/software/coley/recaf/util/SVG.java
+++ b/recaf-ui/src/main/java/software/coley/recaf/util/SVG.java
@@ -2,6 +2,7 @@
 
 import com.github.weisj.jsvg.SVGDocument;
 import com.github.weisj.jsvg.parser.SVGLoader;
+import com.github.weisj.jsvg.parser.LoaderContext;
 import jakarta.annotation.Nonnull;
 import jakarta.annotation.Nullable;
 import javafx.scene.Node;
@@ -33,7 +34,7 @@ public class SVG {
 	static {
 		EMPTY_DOCUMENT = new SVGLoader()
 				.load(new ByteArrayInputStream(("<svg width=\"0\" height=\"0\" xmlns=\"http://www.w3.org/2000/svg\">" +
-						"</svg>").getBytes(StandardCharsets.UTF_8)));
+						"</svg>").getBytes(StandardCharsets.UTF_8)), null, LoaderContext.createDefault());
 	}
 
 	/**
@@ -111,7 +112,7 @@ public static Node ofFile(@Nonnull String path, int width, int height,
 		SVGDocument document = DOCUMENT_CACHE.computeIfAbsent(path, p -> {
 			try (InputStream is = ResourceUtil.resource(path)) {
 				SVGLoader loader = new SVGLoader();
-				return loader.load(is);
+				return loader.load(is, null, LoaderContext.createDefault());
 			} catch (Exception ex) {
 				return EMPTY_DOCUMENT;
 			}
